src/site/_includes/components/navigation/macro.njk

```diff
@@ -0,0 +1,15 @@
+{% macro navigation(params) %}
+<nav class="js-guidance-navigation guidance-navigation" aria-label="Primary">
```

Comment:
Primary doesn't seem like a good label here – probably best to do what they've done on `https://design-system.service.gov.uk/` – `aria-labelledby` and

```html
<h2 class="govuk-visually-hidden" id="guidance-navigation-heading">Menu</h2>
```